File: lua/raindrops/raindrops.lua
return function(n)
  local ret = ''
  if n % 3 == 0 or n % 5 == 0 or n % 7 == 0 then

    if n % 3 == 0 then
      ret = ret .. 'Pling'
    end

    if n % 5 == 0 then
      ret = ret .. 'Plang'
    end

    if n % 7 == 0 then
      ret = ret .. 'Plong'
    end

    return ret
  else
    return tostring(n)
  end
end

File: javascript/raindrops/raindrops.js
//
// This is only a SKELETON file for the 'Raindrops' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const convert = (num) => {
	const by3 = num % 3 == 0;
	const by5 = num % 5 == 0;
	const by7 = num % 7 == 0;
	if (!by3 && !by5 && !by7) {
		return num.toString();
	} else {
		let result = "";
		if (by3) {
			result += "Pling";
		}
		if (by5) {
			result += "Plang";
		}
		if (by7) {
			result += "Plong";
		}
		return result;
	}
};
